lib/openstax/rescue_from/logger.rb

```diff
@@ -0,0 +1,31 @@
+module OpenStax
+  module RescueFrom
+    class Logger
+      attr_reader :proxy
+
+      def initialize(proxy)
+        @proxy = proxy
+      end
+
+      def record_system_error!(prefix = "An exception occurred")
+        Rails.logger.error("#{prefix}: #{proxy.name} [#{proxy.error_id}] " +
```

Comment:
Sorry if I confused the discussion before -- `config.logger.error` is fine for me.  I just wanted to take `system_` out of the name of the variable because it didn't really add much and the configured logger doesn't need to be a "system" logger.